storage/innobase/include/univ.i

```diff
@@ -65,6 +65,12 @@ component, i.e. we show M.N.P as M.N */
 	IB_TO_STR(INNODB_VERSION_MAJOR) "."		\
 	IB_TO_STR(INNODB_VERSION_MINOR) "/en/"
 
+/* Used for progress reports and to tell systemd to not timeout */
+#define INNODB_REPORT_INTERVAL 15
+/* How far ahead should we tell systemd the timeout is based on a
+INNODB_REPORT_INTERVAL (time converted to usec for systemd) */
+#define INNODB_EXTEND_TIMEOUT_INTERVAL (INNODB_REPORT_INTERVAL * 2 * 1000000)
```

Comment:
Start comments with `/**` for Doxygen. Specify that `INNODB_REPORT_INTERVAL` is in seconds.